mytransactions.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals 
+SET species = 'digimon'
+WHERE name LIKE '%mon'; 
+UPDATE animals 
+SET species = 'pokemon'
+WHERE species is NULL; 
+COMMIT;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Verify%20that%20change%20was%20made%20and%20persists%20after%20commit.), Kindly verify that change was made and persists after commit.

Reply:
Also, this was verified and captured in the screenshot of the second transaction.